src/utils/calldata/parser/index.ts

```diff
@@ -20,3 +20,12 @@ export function getAbiVersion(abi: Abi) {
   if (isCairo1Abi(abi)) return 1;
   return 0;
 }
+
+export function isNoConstructorValid(
```

Comment:
this name is a bit strange to me 🤔

Reply:
Yea, agree.
Full name will be: 'Is it valid that the constructor is not part of the abi in this procedural case?'